subsys/bluetooth/host/smp.c

```diff
@@ -264,6 +265,37 @@ static bool sc_supported;
 static const u8_t *sc_public_key;
 static K_SEM_DEFINE(sc_local_pkey_ready, 0, 1);
 
+int bt_smp_get_pairing_peer_info(const struct bt_conn *conn, struct bt_smp_pairing *info)
+{
+	int i;
+
+	BT_DBG("conn %p handle %u", conn, conn->handle);
+
+	for (i = 0; i < ARRAY_SIZE(bt_smp_pool); i++) {
+		struct bt_smp *smp = &bt_smp_pool[i];
+
+		if (smp->chan.chan.conn != conn) {
```

Comment:
You should probably add a check for preventing this API `conn == NULL` (i.e. an -EINVAL return to the beginning of it) since without it this for-loop would try to peek into invalid entries.

Reply:
Thanks, just fixed this along with the formatting!